src/middlewares/index.js

```diff
@@ -59,17 +59,19 @@ export const validateId = key => (req, res, next) => {
 export const validateZaboId = validateId("zaboId");
 
 export const isAdmin = ash(async (req, res, next) => {
-  const { sid } = req.decoded;
-  const self = await User.findOne({ sso_sid: sid });
-  const adminUser = await AdminUser.findOne({ user: self._id });
-  if (!adminUser) {
-    return res.status(403).json({
-      error: "Permission Denied",
+  const { isAdmin } = req.session;
+  const { adminId } = req.session;
+  if (isAdmin) {
+    const adminUser = await AdminUser.findOne({ user: adminId }).populate(
+      "user",
+    );
+    req.adminUser = adminUser;
+    next();
+  } else {
+    return res.status(404).json({
+      error: "not administer",
```

Comment:
```suggestion
      error: "not administrator",
```
오류 메시지를 요렇게 바꿔보는 것은 어떨까요?